web/tests/narrow_local.test.cjs

```diff
@@ -2,8 +2,18 @@
 
 const assert = require("node:assert/strict");
 
-const {zrequire} = require("./lib/namespace.cjs");
-const {run_test} = require("./lib/test.cjs");
+const {mock_esm, zrequire} = require("./lib/namespace.cjs");
+const {run_test, noop} = require("./lib/test.cjs");
+
+function ConversationParticipants() {
+    return {
+        add_from_messages: noop,
+    };
+}
+
+mock_esm("../src/conversation_participants.ts", {
+    ConversationParticipants,
+});
```

Comment:
Do we need to mock this? I'd kinda have expected that the real code (with no data inside) would work cleanly for these code paths.

Reply:
Okay, changed to mock `people.maybe_get_user_by_id` instead.